airflow/providers/microsoft/azure/provider.yaml

```
@@ -298,3 +298,17 @@ logging:
 
 extra-links:
   - airflow.providers.microsoft.azure.operators.data_factory.AzureDataFactoryPipelineRunLink
+
+config:
+  azure_remote_logging:
+    description: |
+      Configuration that needs to be set for enable remote logging in Azure Blob Storage
+    options:
+      remote_wasb_log_container:
+        description: |
+          WASB storage container where the remote logs will be stored.
+          The container should exist.
+        version_added: 7.1.0
```

```suggestion
        version_added: 8.0.0
```

Next version is `8.0.0`